layouts/case-studies/single.html

```diff
@@ -0,0 +1,27 @@
+{{ define "title" }}
```

Comment:
The client's name is displayed at the top in the header, so it felt redundant to display it again above `Problem`.